editions/tw5.com/tiddlers/filters/stringify_Operator.tid

```diff
@@ -18,19 +18,16 @@ The following substitutions are made:
 |!Character |!Replacement |!Condition |
 |`\` |`\\` |Always |
 |`"` |`\"` |Always |
-|Carriage return (0x0d) |`\r` |Always |
+|`'` |`\'` |Always |
 |Line feed (0x0a) |`\n` |Always |
-|Backspace (0x08) |`\b` |Always |
-|Form field (0x0c) |`\f` |Always |
-|Tab (0x09) |`\t` |Always |
-|Characters from 0x00 to 0x1f |`\x##` where ## is two hex digits |Always |
+|Carriage return (0x0d) |`\r` |Always |
+|Characters from 0x00 to 0x1f, except listed above |`\x##` where ## is two hex digits |Always |
 |Characters from 0x80 to 0xffff|`\u####` where #### is four hex digits |If `rawunicode` suffix is not present (default) |
 |Characters from 0x80 to 0xffff|<<.from-version "5.1.23">> Unchanged |If `rawunicode` suffix is present |
 
-<<.from-version "5.1.23">> If the suffix `rawunicode` is present, Unicode characters above 0x80 (such as ß, ä, ñ or 🎄) will be passed through unchanged. Without the suffix, they will be substituted with `\u` codes, which was the default behavior before 5.1.23.
-
-<<.note """Characters outside the Basic Multilingual Plane, such as 🎄 and other emojis, will be encoded as a UTF-16 surrogate pair, i.e. with two `\u` sequences.""">>
+<<.from-version "5.1.23">> If the suffix `rawunicode` is present, Unicode characters above 0x80 (such as ß, ä, ñ or 🎄) will be passed through unchanged. Without the suffix, they will be substituted with `\u` codes, which was the default behavior before 5.1.23. Characters outside the Basic Multilingual Plane, such as 🎄 and other emojis, will be encoded as a UTF-16 surrogate pair, i.e. with two `\u` sequences.
```

Comment:
No critique here, just a question for the core team.  How long do we keep these "New in 5.1.23" around?  There's been the entire `5.2.x` train as well as `5.3.0` and `5.3.1`, and this would likely appear in `5.3.2`.  Is there some official shelf-life?

Reply:
Good question! When we introduced the version tags, I was indeed thinking that we'd have some sort of statute of limitations where we'd drop the older ones. We could do it crudely by searching and replacing in an editor, but I'd prefer to update the macro so that it parses the version number and doesn't produce any output if the version is less than some threshold.